docs/Lexicon.md

```diff
@@ -36,6 +36,10 @@
 * DAG  
   Directed Acylic Graph  
 
+* DMA  
+  Direct Memory Access
+  Copy data to/from memory without occupying CPU time
```

Comment:
if you look at rendering that's on the same line as "Direct Memory Access", looks a bit messy. Can you fix this.

Reply:
fixed